pkg/build/controller/controller.go

```diff
@@ -134,6 +132,40 @@ func (bc *BuildController) nextBuildStatus(build *buildapi.Build) error {
 	return nil
 }
 
+// resolveImageStreamReference looks up the ImageStream[Tag/Image] and converts it to a
+// docker pull spec that can be used in an Image field.
+func (bc *BuildController) resolveImageStreamReference(to *kapi.ObjectReference, namespace string) (string, error) {
```

Comment:
Slightly different, as at the time of writing From used IST, ISI, DI, whereas To used IS, IST, DI. Although I'm not sure why we don't allow IS in the From field, but we can't allow having ISI in the To field.